css/custom.css

```diff
@@ -1,6 +1,7 @@
 @charset "UTF-8";
```

Comment:
is `@import` a scss thing? I have a feeling it's not valid CSS. Can you confirm that.

Reply:
Import url is not a scss thing, however people are saying it takes a hit on the speed at which your site runs so ill use a different method.